### DISCUSSION TITLE
How to achieve multiple category API

### DISCUSSION THREAD

**Discussion by ranamanas** [fastapi/fastapi#6547] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
app = FastAPI()
app.title = "Test data setup for mobile tests"
app.description = "Helper service to setup test data for mobile automation"
app.version = "2.0.0"
app.debug = True


def configure():
    app.include_router(home.router)
    app.include_router(userlingo.router)
    app.include_router(core_services.router)



### Description
- userlingo and core_services are 2 different category like dash/syllabus and under this category have different API's
want to configure multiple category and each category should have different endpoint. Currently, i have done in Flask and i am moving from Flask to FastAPI so could you please let me know - how to achieve this ?
Note : Attached screenshot which i did in Flask+python
cc: @tiangolo 


### Operating System

macOS

### Operating System Details

Mac OS

### FastAPI Version

0.68

### Python Version

python version is 3.9

### Additional Context


<img width="1023" alt="Screen Shot 2021-08-18 at 4 57 33 PM" src="https://user-images.githubusercontent.com/42813960/129987727-02f035d9-8501-4847-8aaa-250cd444ce65.png">

**Comment by ranamanas** [comment]
<img width="1023" alt="Screen Shot 2021-08-18 at 4 57 33 PM" src="https://user-images.githubusercontent.com/42813960/129987770-f0e74a80-15ee-4137-bc2c-88267b7f7dba.png">

**Comment by YuriiMotov** [comment, marked as answer]
Using Tags: https://fastapi.tiangolo.com/tutorial/bigger-applications/#add-some-custom-tags-responses-and-dependencies
